src/pages/Sessions.tsx

```diff
@@ -144,6 +156,19 @@ const Sessions = () => {
     return classObj !== undefined ? classObj.families : [];
   };
 
+  useEffect(() => {
+    setIsTakingAttendance(false);
+  }, [isOnAttendanceView]);
```

Comment:
ideally attendance will be also locked when users switch around class tabs - can we add a dependency to listen for that?

Reply:
ok i added `classTabIndex` to the things it listens for...... i hope that's okay 😳

Reply:
perfecto!!!